addon-test-support/@ember/test-helpers/setup-application-context.ts

```diff
@@ -173,7 +183,7 @@ export function currentRouteName(): string {
 
   let router = context.owner.lookup('router:main');
 
-  return get(router, 'currentRouteName');
+  return get(router, 'currentRouteName') as string;
```

Comment:
Should this also be an assertion?

Reply:
Not unreasonable. Really, it's only here till we drop support for older versions, at which point we can just do `router.currentRouteName`. I'll update anyway. Also the assertions about the router broke things, though; which means I asserted the wrong thing apparently!